compiler/modules/types/xml.bal

```diff
@@ -24,45 +24,36 @@ const int XML_PRIMITIVE_RO_MASK = XML_PRIMITIVE_NEVER | XML_PRIMITIVE_RO_SINGLET
 const int XML_PRIMITIVE_RW_MASK = XML_PRIMITIVE_ELEMENT_RW | XML_PRIMITIVE_PI_RW | XML_PRIMITIVE_COMMENT_RW;
```

Comment:
Not sure what to do about XML type now. It's not currently general enough to work with cells.

I see there is still some `_RO_`s and `_RW_`s left.  Let's discuss this.

Reply:
We have defined all primitives as follows.
```bal
const int XML_PRIMITIVE_ALL_MASK = XML_PRIMITIVE_RO_MASK | XML_PRIMITIVE_RW_MASK;
```

`XML_PRIMITIVE_RW_MASK` is to represent only the RW parts i.e. `XML_PRIMITIVE_ELEMENT_RW | XML_PRIMITIVE_PI_RW | XML_PRIMITIVE_COMMENT_RW;` ?